tests/boards/sr/v4/test_power_board.py

```diff
@@ -181,3 +183,15 @@ def test_power_board_wait_start() -> None:
     pb.wait_for_start_flash()
 
     assert pb._run_led.state
+
+
+def test_output_mutability() -> None:
+    """
+    Test the mutability of Motors.
```

Comment:
The docstrings of `test_output_mutability()` and `test_servo_mutability()` both refer to motors instead of outputs and servos, respectively.

Reply:
de91bd3